eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -983,13 +983,15 @@ protected function copySubtree($actionMap, $oldParentAliasId, $newParentAliasId,
 
             $alreadyGeneratedAliases[] = $identifier;
 
-            $alreadyGeneratedAliases = array_merge(
-                $alreadyGeneratedAliases,
-                $this->copySubtree(
-                    $actionMap,
-                    $oldParentAliasId,
-                    $row['id'],
-                    $alreadyGeneratedAliases
+            $alreadyGeneratedAliases = array_unique(
+                array_merge(
+                    $alreadyGeneratedAliases,
+                    $this->copySubtree(
+                        $actionMap,
+                        $oldParentAliasId,
+                        $row['id'],
+                        $alreadyGeneratedAliases
+                    )
```

Comment:
```suggestion
            $alreadyGeneratedAliases = $this->copySubtree(
                $actionMap,
                $oldParentAliasId,
                $row['id'],
                $alreadyGeneratedAliases
            );
```

Reply:
Done in https://github.com/ezsystems/ezpublish-kernel/pull/3128/commits/2ec56892d39794a91342e9914aa37a525123a6b5